config/buildspec_profiler_sagemaker_pytorch_1_5_0_integration_tests.yml

```diff
@@ -0,0 +1,41 @@
+# Build Spec for AWS CodeBuild CI
+
+version: 0.2
+env:
+  variables:
+    override_smdebug: "true" # should we install the latest version of smdebug or use the existing version of smdebug?
```

Comment:
maybe this variable can be renamed?

The comment seems to suggest that toggling this variable changes the version of smdebug involved.

Reply:
Sure, can rename to `override_smdebug_version`

Reply:
Done.